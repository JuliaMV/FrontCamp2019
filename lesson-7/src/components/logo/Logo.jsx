import React from 'react';
import css from './Logo.module.css';

const Logo = () => (
  <a href="./index.html">
    <span>netlix</span>roulette
  </a>
);

export default Logo;
