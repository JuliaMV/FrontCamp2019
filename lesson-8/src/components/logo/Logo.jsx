/* eslint-disable react/jsx-one-expression-per-line */
import React from 'react';

import css from './Logo.module.css';

const Logo = () => (
  <a className={css.logo} href="./index.html">
    netlix<span>roulette</span>
  </a>
);

export default Logo;
