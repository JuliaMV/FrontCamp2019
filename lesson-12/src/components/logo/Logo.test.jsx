import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import { create } from 'react-test-renderer';
import Logo from './Logo';

describe('Logo component', () => {
  test('it matches the snapshot', () => {
    const component = create(<Router><Logo /></Router>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
