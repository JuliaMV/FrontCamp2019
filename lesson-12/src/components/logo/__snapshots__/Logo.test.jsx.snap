// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component it matches the snapshot 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  netlix
  <span>
    roulette
  </span>
</a>
`;
